src/main/java/redis/clients/jedis/HighLevelBinaryJedisClusterCommands.java

```diff
@@ -0,0 +1,5 @@
+package redis.clients.jedis;
+
+public interface HighLevelBinaryJedisClusterCommands {
```

Comment:
Why another interface?. This command doesn't seem high level it just checks that the key contains a hash tag. 

:-1:  @HeartSaVioR ?